Content.Shared/Damage/Systems/DamageableSystem.cs

```diff
@@ -191,6 +192,17 @@ public void DamageChanged(EntityUid uid, DamageableComponent component, DamageSp
             if (before.Cancelled)
                 return null;
 
+            // Hit anything the target is wearing, too. But on recursive hits, only pass positive values. Healing the
+            // player shouldn't also fix their hardsuit.
+            // TODO: The player drinking acid shouldn't deal heat damage to worn items. However, DamageSystem doesn't
+            // currently support this kind of modeling.
+            var dp = DamageSpecifier.GetPositive(damage);
```

Comment:
goes for the stuff in barotrauma system too, but modifications to upstream files should be enclosed with `// ES START` and `// ES END` comments for tracking where changes come. It just helps with resolving any potential merge conflicts.

Reply:
Sure.